app/src/Theme/Layout.ts

```diff
@@ -1,10 +1,13 @@
+/**
+ * @file reusable layouts for common components
+ */
 import { StyleSheet } from 'react-native'
 import { ThemeLayout, ThemeVariables } from '@/Theme/theme.type'
 
 /**
  *
- * @param Theme can be spread like {Colors, NavigationColors, Gutters, Layout, Common, ...args}
- * @return {*}
+ * @param {ThemeVariables} ThemeVariables can be spread like {Colors, NavigationColors, Gutters, Layout, Common, ...args}
+ * @returns {ThemeVariables} Different theme layouts
```

Comment:
Should the return type here actually be `ThemeLayout`?  At the end of line 12 the Typescript type for what the function returns is that